app/models/competition.rb

```diff
@@ -1169,8 +1169,8 @@ def has_event_change_deadline_date?
 
   # can registration edits be done right now
   # must be allowed in general, and if the deadline field exists, is it a date and in the future
-  def registration_edits_allowed?
-    self.allow_registration_edits &&
+  def registration_edits_permitted?
```

Comment:
Maybe bake `currently` into the name since we're checking against `DateTime.now`?